h2o-algos/src/main/java/hex/deeplearning/DeepLearningModel.java

```diff
@@ -2305,5 +2305,10 @@ public boolean isFeatureUsedInPredict(String featureName) {
     int featureIdx = ArrayUtils.find(varImp()._names, featureName);
     return featureIdx != -1 && (double) varImp()._varimp[featureIdx] != 0d;
   }
+
+  @Override
+  public boolean isDistributionHuber() {
+    return _parms._distribution == DistributionFamily.huber || get_params()._distribution == DistributionFamily.huber;
```

Comment:
I think this should call super.isDistributionHuber, eg. super.isDistributionHuber || get_params()._ ...